### PR TITLE
adds config.js to api

### PR DESCRIPTION
**Pull request by lbai001**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// this file controlls whether forum apis are going to be used
+// default pushkin doest not have forum apis
+// feel free to change the settings
+
+module.exports = CONFIG = {
+  forum: false
+};
```
Comment (RobAWilkinson) [inline config.js]: Add this comment to readme.

File: forum.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+const express = require('express');
+
+module.exports = (rpc, conn, dbwrite) => {
+  const router = new express.Router();
+  router.get('/getAllForumPost', (req, res, next) => {
+    var rpcInput = {
+      method: 'allForumPost',
+      params: []
+    };
+    const channelName = 'forum_rpc_worker';
```
Comment (RobAWilkinson) [inline forum.js]: can we bring this up to line 2 since it stays the same

File: forum.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      params: [
+        {
+          auth0_id: req.body.auth0_id,
+          post_content: req.body.post_content,
+          stim_id: req.body.stim_id,
+          post_subject: req.body.post_subject,
+          created_at: req.body.created_at
+        }
+      ]
+    };
```
Comment (RobAWilkinson) [inline forum.js:30]: fyi if you need to do this in the future, theres also `lodash#pluck` if you have lodash.

File: server.js [hunk 4]
```diff
@@ -30,39 +30,10 @@ amqp.connect(process.env.AMPQ_ADDRESS, function(err, conn) {
...
-    const channelName = 'forum_rpc_worker';
-    return rpc(conn, channelName, rpcInput)
-      .then(data => {
-        res.json(data);
-      })
-      .catch(next);
-  });
-
+  if (CONFIG.forum) {
+    const forumController = require('./forum')(rpc, conn, dbWrite);
```
Comment (RobAWilkinson) [inline server.js:34]: I <3 conditional requires.

Comment (lbai001) [reply server.js:34]: use a real heart please

File: forum.js (hunk 3 above)
Comment (lbai001) [reply forum.js:30]: seems like lodash removed _.pluck cause of the similarity between _.pluck and _.map
now we could use _.map(obj, key) to do the same thing, but acknowledged

Comment (RobAWilkinson) [reply forum.js:30]: ah interesting, good catch

File: server.js (hunk 4 above)
Comment (RobAWilkinson) [reply server.js:34]: ❤️

**Pull request merged**